src/components/Search/Filters/FilterOptions.tsx

```diff
@@ -0,0 +1,118 @@
+import { useRouter } from 'next/router'
+import { ReactElement } from 'react'
+import {
+  FilterByTypeOptions,
+  ResultsPerPageOptions,
+  SortByOptions,
+  SortDirectionOptions
+} from '../../../models/SortAndFilters'
+import styles from './FilterOptions.module.css'
+import classNames from 'classnames/bind'
+
+const cx = classNames.bind(styles)
+
+export type TypeKeys = keyof typeof filterTypeOptions
+
```

Comment:
suggestion(non-blocking): This component is probably a good place to implement the interfaces or we can also move this into the `types` directory.